generators/app/templates/core/{{cookiecutter.bot_name}}/README.md

```diff
@@ -0,0 +1,61 @@
+# CoreBot
+
+Bot Framework v4 core bot sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to:
+
+- Use [LUIS](https://www.luis.ai) to implement core AI capabilities
+- Implement a multi-turn conversation using Dialogs
+- Handle user interruptions for such things as `Help` or `Cancel`
+- Prompt for and validate requests for information from the user
+
+## Prerequisites
+
+This sample **requires** prerequisites in order to run.
+
+### Overview
+
+This bot uses [LUIS](https://www.luis.ai), an AI based cognitive service, to implement language understanding.
+
+### Install Python 3.7
```

Comment:
>Python 3.7 [](start = 12, length = 10)

I'm still not sure if we're 3.7 or 3.6 here. ;)

Reply:
Solved